webflux/src/test/java/org/springframework/security/web/server/DefaultServerRedirectStrategyTests.java

```diff
@@ -46,16 +44,14 @@
 	private DefaultServerRedirectStrategy strategy =
 		new DefaultServerRedirectStrategy();
 
-	private AuthenticationException exception = new AuthenticationCredentialsNotFoundException("Authentication Required");
-
 	@Test(expected = IllegalArgumentException.class)
 	public void sendRedirectWhenLocationNullThenException() {
-		this.strategy.sendRedirect(this.exchange, (URI) null);
```

Comment:
This is intentional so that things don't break if an overloaded method was added